notch8/atla_digital_library
# Fix Media Viewer for videos

co-author: @kirkkwang 

# Summary
UV was showing just a black square for videos. the issue is that the file duration was not being set. this file fixes them by pulling in some parts of the Dockerfile from hyku, as well as changes kirk made in British Libraries to make the av_gem work

# Related
#195 

# Screenshot
<img width="1309" alt="image" src="https://user-images.githubusercontent.com/73361970/232627026-86e4cb24-7084-4457-a291-b32e78cb2f2e.png">

# Video
https://share.getcloudapp.com/Qwu8w59P

# Acceptance Criteria
- [ ] As a user, I can view videos in the UV Viewer

# Testing Instructions
1. Create a work with worktype MovingImage & check the "Use IIIF Viewer" option
2. Upload a video file - example file you can use: https://user-images.githubusercontent.com/73361970/232627418-8ef69351-6e74-428d-a508-2fb63aa00dec.mp4
3. Wait for the file to upload

4. View the work show page & make sure you can see the video you uploaded in the viewer.
